### PR TITLE
Update meta tag description

### PR DESCRIPTION
**Pull request by DevinLeamy**
copilot:poem

### Why
- Fixes GI-4857.
- _Twitter cards use the meta tag `property="og:description"` for descriptions._

### how

copilot:walkthrough

### CODE REVIEW

**Comment by DevinLeamy** [comment]
In `StaticPage` we could also just set a default description that always appears in the meta tags, rather can only having a description when one is provided. 

```ts
<meta
  property="og:description"
  content={openGraphMetadata.description ?? "DEFAULT"}
/>
```

File: src/pages/splash.tsx [hunk 1]
```diff
@@ -211,12 +211,14 @@ export const SplashPage: React.FunctionComponent<{
 
   const [playYoutubeTrailer, setPlayYoutubeTrailer] = useState(false);
   return (
-    <>
-      <Head>
-        <meta name="theme-color" content="#1C0D28" />
-        <meta name="viewport" content="width=device-width, initial-scale=1" />
-        <title>Biomes — Join the community shaping a new world</title>
-      </Head>
+    <StaticPage
```
Comment (turtlesoupy) [inline src/pages/splash.tsx:214]: Can you check the root page too? I think it is index.tsx — I believe it does something different than splash

Comment (DevinLeamy) [reply src/pages/splash.tsx:214]: Should be fine, but it's been updated!
```ts
<RootErrorBoundary>
    <BiomesHeadTag description="Biomes is an open source sandbox MMORPG built for the web using web technologies" />
    <SplashPage
      onLogin={() => {
        window.location.href = "/at";
      }}
    />
  </RootErrorBoundary>
  ```

File: src/pages/index.tsx [hunk 2]
```diff
@@ -54,7 +54,7 @@ export const BiomesHeadTag: React.FunctionComponent<BiomesHeadTagProps> = (
 export default function Index({}: {}) {
   return (
     <RootErrorBoundary>
-      <BiomesHeadTag />
+      <BiomesHeadTag description="Biomes is an open source sandbox MMORPG built for the web using web technologies" />
```
Comment (turtlesoupy) [inline src/pages/index.tsx:57]: Can you make this the default in that head tag?

**Review by turtlesoupy — APPROVED** [review]

File: src/pages/index.tsx (hunk 2 above)
Comment (DevinLeamy) [reply src/pages/index.tsx]: Done.

**Pull request merged**